apps/saleor/src/index.ts

```diff
@@ -0,0 +1,65 @@
+import { DialogExtensionSDK, FieldExtensionSDK } from 'contentful-ui-extensions-sdk';
+import { setup, renderSkuPicker } from 'shared-sku-app';
+import { dialogConfig, DIALOG_ID, SKUPickerConfig, strings } from './constants';
+
+import PaginatedFetcher from './PaginatedFetcher';
+import { ClientConfig, Identifiers } from './types';
+
+const makeCTA = (fieldType: string) => {
+  return fieldType === 'Array' ? strings.selectProducts : strings.selectProduct;
+};
+
+const validateParameters = (parameters: ClientConfig): string | null => {
+  if (parameters.apiEndpoint.length < 1) {
+    return 'Missing API Endpoint';
+  }
+
+  return null;
+};
+
+const createContainer = () => {
```

Comment:
Thought it would be more explicit to specify the id being used in the js file instead of relaying on the one in html, but it's not a big issue so can change if you feel like it's needed

Reply:
I don't think it's hugely important, I just wanted to check if there was a particular reason for this. I'm happy with this approach too 👍